### PR TITLE
docs: translate Children

### LINKED ISSUE #546: `Children`
Page: https://id.react.dev/reference/react/Children

### PR DESCRIPTION
**Pull request by chez14**
Fixes #546 and closes #573.

## Description

This PR adds translation to "Children" page, continuing @ilham89 effort.

Translate the Suspense page.
Page URL:
- 🌍: https://react.dev/reference/react/Children
- 🇮🇩: https://id.react.dev/reference/react/Children

### CODE REVIEW

**Comment by chez14** [comment]
I'm having problem with these terms:
- Traverse → Jelajah.
  - Traversal → Penjelajahan.
  - Traversed → Dijelajahi.

**Comment by chez14** [comment]
Hi @resir014 and @mazipan,

This PR is ready for review, can you please review it? Let me know if you have any feedback or questions.

In addition to that, I have hard time translating some terms. The list of terms can be checked on my [earlier comment](https://github.com/reactjs/id.react.dev/pull/609#issuecomment-1694339494).

Thank you so much!

File: src/content/reference/react/Children.md [hunk 1]
```diff
@@ -27,44 +27,44 @@ const mappedChildren = Children.map(children, child =>
...
+#### Kembalian {/*children-count-returns*/}
 
-The number of nodes inside these `children`.
+Jumlah simpul dalam `children` ini.
 
-#### Caveats {/*children-count-caveats*/}
+#### Catatan Penting {/*children-count-caveats*/}
 
-- Empty nodes (`null`, `undefined`, and Booleans), strings, numbers, and [React elements](/reference/react/createElement) count as individual nodes. Arrays don't count as individual nodes, but their children do. **The traversal does not go deeper than React elements:** they don't get rendered, and their children aren't traversed. [Fragments](/reference/react/Fragment) don't get traversed.
+- Simpul kosong (`null`, `undefined`, dan *Boolean*), *string*, angka, dan [elemen React](/reference/react/createElement) dihitung sebagai simpul individu. Larik tidak dihitung sebagai simpul individu, tetapi anak-anaknya dihitung sebagai simpul individu. ***Traversal* tidak masuk lebih dalam dari elemen React:** mereka tidak di-*render*, dan anak-anaknya tidak di-*traverse*. [Fragmen](/reference/react/Fragment) tidak di-*traverse*.
```
Comment (mazipan) [inline src/content/reference/react/Children.md:61]: array = senarai, sesuai dengan apa yang kita sepakati di laman glossary

Comment (chez14) [reply src/content/reference/react/Children.md:61]: Ah! Sorry, I forgot to crosscheck this term. I'll update accoringly.

**Comment by chez14** [comment]
Hi @mazipan,

I've updated the term to senarai. Can you please re-review this PR again?

Thank you so much!

**Review by mazipan — APPROVED** [review]

**Pull request merged**